client/components/domains/register-domain-step/index.jsx

```diff
@@ -337,9 +337,26 @@ class RegisterDomainStep extends Component {
 		return strippedHostname ?? hostname;
 	}
 
+	getInitialQueryInAIFlow() {
+		if ( this.props.flowName !== 'ai-site-builder' ) {
```

Comment:
Let's use the const from the onboarding package here. These consts help us a lot if we decide to remove the flow, as the const would lead us to all the conditions tied to it.

Reply:
https://github.com/Automattic/wp-calypso/pull/100948/commits/82ef7eb326213c02f38296ab314a690ba9e8b7cf